server/src/model/blog.js

```diff
@@ -1,5 +1,5 @@
 import mongoose from 'mongoose'
-import Tag from './tag.js'
+import Tag from './tags.js'
```

Comment:
follow the proper linting method.  enter after function change for making reading easier